include/echoFunctions.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+
+function echoHeader($Title, $PageName) {
+
+	?>
+	<script src="/include/jquery.js"></script>
+	<?php
+
+	echo "
+		<head>
+			<title>$Title</title>
+			<link rel='stylesheet' href='style.css'/>
+			<link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Quicksand'>
+			<link href='https://fonts.googleapis.com/css?family=Oxygen' rel='stylesheet'>
+		</head>
+
+";
+?>
+
+<script type='text/javascript'>
+
+
+function intakeVote(eventID, userID, direction){
```

Comment:
Yup, just from the session. No one should be able to vote if they're not logged in (otherwise you couldn't count the vote correctly) and if they're logged in, their UserId will be in the session.

From a security standpoint, you should never trust a user to tell you who they are aside from during the login process. If they're passing a UserId to the server, they can forge requests from other users too easily.